### ISSUE TITLE
Error: No acceptable window type found for: 'undefined' when trying to login via github

### ISSUE THREAD

**Issue by chit786** [cypress-io/cypress#1093]
- Operating System: macOS Sierra
- Cypress Version: 1.3.0
- Browser Version:

### Is this a Feature or Bug?
BUG

### Current behavior:
github login window do not show

### Desired behavior:
github login window should be shown

### How to reproduce:
 - install cypress as --save-dev
 - Make sure no user is configured for cypress to be logged in.
 - click Log in
 - Cypress should behave same as 1.2.0 but it fails with error

```
Opening Cypress...
Error: No acceptable window type found for: 'undefined'
```

#### Test code:

```js

```

**Comment by brian-mann** [comment]
Oh wow. Yes this is completely broken. Regression in `1.3.0`. Very surprised no tests caught this, but it happens sometimes. We'll fix this immediately.

**Comment by brian-mann** [comment]
Released in [`1.4.0`](https://docs.cypress.io/guides/references/changelog.html#1-4-0).
